src/app/[locale]/specs/repository/page.mdx

```diff
@@ -125,6 +125,38 @@ When importing CAR files, note that there may existing dangling CID references.
 
 The CARv1 specification is agnostic about the same block appearing multiple times in the same file ("Duplicate Blocks)". Implementations should be robust to both duplication and de-duplication of blocks, and should also ignore any unnecessary or unlinked blocks.
 
+## Repository Diffs
+
+A concept which supports efficient synchronization of data between independent services is "diffs" of repository trees between different revisions. The basic principle is that a repository diff contains all the data (commit object, MST nodes, and records) that have changed between an older revision and the current revision of a repo. The diff can be "applied" to the older mirror of the repository, and the result will be the complete MST tree at the current (newer) commit revision.
+
+Repo diffs can be serialized as CAR files, sometimes referred to as "CAR slices". Some details about diff CAR slices:
+
+- same format, version, and atproto-specific constraints as full repo export CAR files
+    - blocks "should" be de-duplicated by CID (only one copy included), though receiving implementations must be resilient to duplication
+- the root CID indicated in the CAR header (the first element of `roots`) should point to the commit block (which must be included)
```

Comment:
I believe we currently require that the CAR only has 1 root (though we could adapt the implementation)

Reply:
I think this is fine / consistent enough? I think i'd like other folks to handle multiple roots on the "processing a CAR slice" side, but not assume they can have multiple roots on the "creating CAR slices" side, and I think this language achieves that?

IMO the next step would be a bit more flexibility in receiving implementation, but that this won't cause problems going out in the spec this way.